dateparser/date.py

```diff
@@ -175,7 +175,7 @@ def _try_given_formats(self):
         if not self.date_formats:
             return
 
-        return parse_with_formats(self._get_translated_date_with_formatting(), self.date_formats)
+        return parse_with_formats(self._get_translated_date_with_formatting(), list(self.date_formats))
```

Comment:
We used [this check](https://github.com/scrapinghub/dateparser/blob/master/dateparser/date.py#L222) for similar case. May be it is worth to add this group to `dateparser.utils` as `plain_iterable` or something. What do you think?